### PR TITLE
Community features

### PR DESCRIPTION
**Pull request by erickmartins**
Opening a PR up for discussion. Things added:

- PR template
- Code of conduct
- Contributing guidelines

Won't merge this immediately - feedback is more than welcome!

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+and point out this code of conduct (either in public or in private, whatever is
+most appropriate). If you would prefer not to do that, please feel free to
+report to the core developers directly, or ask them for
+advice, in confidence.
+
+Currently, the core developers are:
+
+- Dave Mellert <dave.mellert@jax.org>
+- Erick Ratamero <erick.ratamero@jax.org>
+
```
Comment (mellertd) [inline CODE_OF_CONDUCT.md:119]: I agree we can cut most of the reporting stuff. An email to a core team member is probably enough

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+--------
+
+We are thankful to the groups behind the following documents, from which we
+drew content and inspiration:
+
+- [The SciPy project](https://www.scipy.org/)
+- [The Apache Foundation Code of Conduct](https://www.apache.org/foundation/policies/conduct.html)
+- [The Contributor Covenant](https://www.contributor-covenant.org/version/1/4/code-of-conduct)
+- [Jupyter Code of Conduct](https://github.com/jupyter/governance/tree/master/conduct)
+- [Open Source Guides - Code of Conduct](https://opensource.guide/code-of-conduct/)
```
Comment (mellertd) [inline CODE_OF_CONDUCT.md:176]: we need to reference scikit-image

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+  [open issues on GitHub](https://github.com/TheJacksonLaboratory/ezomero/issues?q=is%3Aopen).
+* The technical detail of the development process is summed up below.
+
+
+Stylistic Guidelines
+--------------------
+
+* Functions should strive to, whenever possible, have arguments and returns in common Python types, or in numpy/scipy types. No passing OMERO objects back and forth!
+
+* If there is a function from the base [OMERO-py API](https://downloads.openmicroscopy.org/omero/5.6.3/api/python/) that does something, we do not need to duplicate it. Wrapping OMERO-py functions for simpler input/output is fine.
```
Comment (mellertd) [inline CONTRIBUTING.md:18]: "If there is a function from the base [OMERO-py API](https://downloads.openmicroscopy.org/omero/5.6.3/api/python/) that does something,"

I might add: "e.g., return an `omero.gateway.ImageWrapper` or similar with `conn.getObject`"

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+
+   * Commit locally as you progress (``git add`` and ``git commit``)
+
+3. To submit your contribution:
+
+   * Push your changes back to your fork on GitHub::
+
+      git push origin add-retrieve-tags
+
+   * Enter your GitHub username and password (repeat contributors or advanced
```
Comment (mellertd) [inline CONTRIBUTING.md:79]: note that GitHub has deprecated password authenticating via git without an access token and will discontinue this functionality by Aug 13, 2021. So we might mention this

Comment (erickmartins) [reply CONTRIBUTING.md]: This change (and the two previous comments) implemented in a new commit

**Pull request merged**